src/restore/index.ts

```diff
@@ -0,0 +1,45 @@
+import * as core from '@actions/core'
+import * as exec from '@actions/exec'
+import * as p from 'path'
+import {checkKey, checkPaths, getCachePath, options} from '../utils/cache'
+
+async function run(): Promise<void> {
+  try {
+    const key = core.getInput('key')
+    const path = core.getInput('path')
+    const cachePath = getCachePath(key)
+
+    checkKey(key)
+    checkPaths([path])
+
+    core.saveState('key', key)
+    core.saveState('path', path)
+    core.saveState('cache-path', cachePath)
+
+    let {stdout, stderr, listeners} = options('', '')
+    await exec.exec(`/bin/bash -c "test -d ${cachePath} ; echo $? `, [], {
+      listeners
+    })
+
+    const cacheHit = stdout === '0' ? 'true' : 'false'
```

Comment:
이거 boolean으로 하는게 어때요? setOutput 할 때만 스트링으로 저장하고요.

```suggestion
    const cacheHit = stdout === '0' ? true : false
```

Reply:
오 그게 훨씬 좋을 것 같아요